sherpa/models/parameter.py

```diff
@@ -644,6 +644,26 @@ def _repr_html_(self):
     __mod__, __rmod__ = _make_binop(numpy.remainder, '%')
     __pow__, __rpow__ = _make_binop(numpy.power, '**')
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        if not method == '__call__':
+            return NotImplemented
+        if hasattr(numpy, ufunc.__name__):
+            name = f"numpy.{ufunc.__name__}"
+        else:
```

Comment:
After your mention of scipy.gamma.__class__ being numpy.ufunc, it strikes me we could try something like

```
if isinstance(ufunc, numpy.ufunc):
    name = <not sure what to do here>
```

but, as indicated above, I'm not sue it would be at all helpful (in fact, I imagine you almost certainly have to be a member of numpy.ufunc to get to this code anyway).

I still find it interesting that I can't seem to identify the module that "owns" a particular numpy.ufunc, but life is too short to worry about this.

Reply:
You've looked and I've looked. I don't think there is a way to find which file instantiated the `np.unfunc` class that we are looking at here. Future versions of numpy might offer more information, but for now this is all we have if we like it or not.